go.mod

```diff
@@ -326,7 +324,7 @@ require (
 
 require (
 	github.com/NYTimes/gziphandler v1.1.1
-	github.com/filecoin-project/boostd-data v0.0.0-00010101000000-000000000000
+	github.com/filecoin-project/boost/extern/boostd-data v0.0.0-20230804101708-7565454b8600
```

Comment:
This is set to a commit from this branch since commits to main do not contain this code yet, nor any specific tag.

We should update this to a commit from main or a tag once one such thing exists. This is the dependency version that will be pulled by other go mods depending on boost.